news/2020-03-20-beatmap-spotlights-winter-2020.md

```diff
@@ -0,0 +1,239 @@
+
+*written by [Yumeno Himiko](https://osu.ppy.sh/users/1806962)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/loving.jpg)](https://osu.ppy.sh/beatmapsets/1080025)
+
+'How on earth did eroge get into spotlights?' may be a question that pops up when first encountering this set. Despite the general stigma against the genre, [Luminate](https://osu.ppy.sh/users/4778689)'s "Loving Trip" does a great job of following the song. 
+
+Slower parts of the song keep things interesting with varied rhythm and patterns, while higher density sections have liberal use of antiflow that contrasts well. This is one of the few sets that I've genuinely had fun playing this year, and I hope you give it a shot as well.
+
+*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/lights.jpg)](https://osu.ppy.sh/beatmapsets/1044462)
+
+[Kukkai](https://osu.ppy.sh/users/7811952)'s "Lights of Muse" has a certain feeling behind it that is fairly uncommon these days, reminiscent of something that would have easily had a play count in the hundred thousand if it was ranked a few years ago. While the use of patterns and rhythm is fairly simple, the whole set is at a consistent level of fun that is hard to come by.
+
+*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/sweet.jpg)](https://osu.ppy.sh/beatmapsets/738169)
+
+[Sorcerer](https://osu.ppy.sh/users/1913190) and [ZiRoX](https://osu.ppy.sh/users/200768) has something sweet to offer in this season's Spotlights!
+
+Starting from the bottom we have Cup, Salad and Platter all made by Sorcerer, and they lead the player through the learning stages of the streams and how to control the jumps. If you can already get full combo on the Salad then the next step is to learn how to handle the Hyperjumps and time your normal jumps at the end of the streams in Platter. Very well designed difficulty curve.
+
+Next up we have two Rain to choose from, one from the mapper of the previous three and one guest difficulty from ZiRoX. Dispite their similar star ratings there is a clear difference in them. ZiRoX decided to use less Hyperjump during the stream parts but instead use stair patterns, which are not necessarily easier to handle though. Meanwhile Sorcerer decided to step up from Platter and make his Hyperjump streams even more deadly for the player with more experience.
+
+And for last we have a Sweet Overdose, which is not really that sweet if you take a peek inside. It raises to jump stream pattern into a whole new level where the players have to face the true evil in case they dare to challenge it. The sweetness flew away and only Rain remained, raining fruits and Hypers everywhere. Prepare your umbrella and get ready to catch 'em all!
```

Comment:
in general less description about what the set is and more description about the themes/techniques it explores is better

Reply:
That is not easy to do when the set is all about one technique and stays consistent though. I am not sure if this one is better either.
```suggestion
Next up we have two Rain to choose from, one from the mapper of the previous three and one guest difficulty from ZiRoX. Despite their similar star ratings, there is a clear difference in them. ZiRoX uses less Hyperjump during the stream parts but went for stair patterns and some quick 1/4 slider jump pattern later on. Meanwhile, Sorcerer decided to step up from Platter and make his Hyperjump streams even more deadly for the player with more experience.

And for last we have a Sweet Overdose which has short slider Hyperdashes to follow the melody. It also uses the same reverse slider pattern from rain to temporarily stop the player's movement. The flow is usually straightforward, but sometimes you might find some antiflow pattern. The sweetness flew away and only Raining fruits remained. Prepare your umbrella and get ready to catch 'em all!
```